services/director-v2/src/simcore_service_director_v2/core/application.py

```diff
@@ -66,6 +67,8 @@ def init_app(settings: Optional[AppSettings] = None) -> FastAPI:
     if settings.registry.enabled:
         docker_registry.setup(app, settings.registry)
 
+    dynamic_sidecar.setup_dynamic_sidecar(app)
```

Comment:
MINOR: unnecessary redundancy. I suggest
```python
dynamic_sidecar.setup(app)
```
OR
```python
from dynamic_sidecar import setup_dynamic_sidecar
setup_dynamic_sidecar(app)
```

Reply:
I do not agree. I was using the same style of importing. At most I would rename `setup_dynamic_sidecar` to `setup`.